src/main/java/uk/gov/hmcts/reform/cwrdapi/util/CaseWorkerConstants.java

```diff
@@ -119,5 +119,8 @@ private CaseWorkerConstants() {
     public static final String ROLE_FIELD = "Role";
     public static final String LOCATION_FIELD = "Location";
     public static final String AREA_OF_WORK_FIELD = "Area Of Work";
+    public static final String FILE = "file";
     public static final String AND = "and";
+
+    public static final String UP_MISSING_RESPONSE_BODY = "UP response missing body";
 }
```

Comment:
messages here need to be reviewed to be more appropriate

Reply:
@abhijitdiwan @shreedhar-hmcts @SujeethKumarAri  - these messages need to be reviewed.. can someone take an action to do this?